src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
+          m_vdsp({}),
+          m_vinfo({}),
```

Comment:
m_vinfo is initalized by vorbis_info_init(); which calls memset(vi,0,sizeof(*vi)); so initalizing this here is also redundant.